src/rpiplatesrecognition/db/__init__.py

```diff
@@ -89,3 +91,32 @@ def access_attempts_integrity_check():
             access_attempts = AccessAttempt.query.all()
             assert all(access_attempt.photos_exist() for access_attempt in access_attempts), \
                 "There are access attempts without photos existing, please reinit db with 'flask init-db' or 'flask init-db-debug'"
+
+
+@click.command('init-db-backup')
+@with_appcontext
+def init_db_backup_command():
+    """Command for creating backup of database and processed images"""
+    
+    try:
+        if not os.path.split(os.getcwd())[1] == 'rpi-plates-recognition':
+            raise Exception("Not proper start point directory. Run command in base repo directory.")
+
+        path = 'backup/'
+        if not os.path.isdir(path):
+            os.mkdir(path)
```

Comment:
Raczej wolałbym, żeby było to jedno archiwum, w którym jest równocześnie folder `photos` oraz plik z bazą danych